### DISCUSSION TITLE
How to create a "real time" animation?

### DISCUSSION THREAD

**Discussion by blogdemoi** [gpx-animator/gpx-animator#502] [Q&A] [answered]
Hello,

I have just installed Gpx-animator, but a little lost regarding the settings.

Let's say my .gpx track lasts 20 minutes, how do I render a real time video that lasts 20 minutes. I ask this because I wish to embed the Gpx-animator rendered video in the video rendered by my camera (in a corner of my camera video).

Thank you.

**Comment by McPringle** [comment, marked as answer]
The `Speedup` option controls the speed of the animation. If you set it to `1`, it should render a "real time" video. Maybe it will make sense to reduce the frames per second. Usually GPS devices do not record more than one position per second, so setting `FPS` to `1` should work fine and will make the rendering much faster.

**Reply by blogdemoi** [reply]
Thank you.

**Reply by blogdemoi** [reply]
> Usually GPS devices do not record more than one position per second, so setting FPS to 1 should work fine and will make the rendering much faster.

So, if my Gps app in my phone records a position every 60s I should set the `FPS` setting to 60?

**Reply by McPringle** [reply]
> So, if my Gps app in my phone records a position every 60s I should set the `FPS` setting to 60?

No, the `FPS` is the frames per second for the resulting video. Less than `1 FPS` is not possible. If your device records the position every 60 seconds, GPX Animator will interpolate the frames between. So you should set `FPS` to `1`.

**Reply by blogdemoi** [reply]
Ok.

**Reply by blogdemoi** [reply]
And one last question, the rendered video is too long. So I tried to change the video length (as indicated "Video is very short or long Fine-tune speedup or specify exact total time in seconds." ). However I can't input anything in the line here:

![MOLz660](https://user-images.githubusercontent.com/83965802/222969325-7f571005-6efa-4c41-b0a4-5db1c6b79ec9.png)

It doesn't allow me to input any value.

**Reply by blogdemoi** [reply]
Edit: ah, ok, I found out. Speed up needs to be at 0.

**Comment by McPringle** [comment]
You have set `Speedup` to `200`. That tells GPX Animator to create the animation 200 times faster than recorded. Remove that setting and the input of `Total time` will be enabled. You can only specify one of the two settings, not both.
